webapp/imports/ui/Game.jsx

```diff
@@ -41,7 +41,7 @@ class Game extends Component {
           </div>
           <div className="eight wide column">
             <h2 className="center">Game Information</h2>
-            <GameInfo game={this.props.game} spectatorMode={spectatorMode} readonly={readonly} />
+            <GameInfo game={this.props.game} spectatorMode={spectatorMode} readonly={readonly} currentUser={this.props.currentUser} />
```

Comment:
Are there any props you specifically don't want to pass down to the child component? I'd say use `_.extend()` to add `spectatorMode` and `readonly` to `this.props` without having to write out every single prop attribute.